src/main/java/Duke.java

```diff
@@ -1,10 +1,149 @@
+import Task.Deadline;
+import Task.Event;
+import Task.Task;
+import Task.Todo;
+
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String REGEX_NUM = "\\d+";
+
+    private static final int TASKS_CAPACITY = 100;
+
+    private static final String AT_ARGUMENT = "/at ";
+    private static final String BY_ARGUMENT = "/by ";
+
+    private static final String BYE_COMMAND = "bye";
+    private static final String DONE_COMMAND = "done";
+    private static final String LIST_COMMAND = "list";
+
+    private static final String EVENT_COMMAND = "event ";
+    private static final String DEADLINE_COMMAND = "deadline ";
+    private static final String TODO_COMMAND = "todo ";
+
+    private static final String PROMPT = "> ";
+
+    private static int taskCount = 0;
+    private static Task[] tasks = new Task[TASKS_CAPACITY];
+
+    private static void printPrompt() {
+        System.out.print(PROMPT);
+    }
+
+    private static void printResponse(String response) {
+        System.out.printf("  %s\n", response);
+    }
+
+    private static void printTasks() {
+        if (taskCount == 0) {
+            printResponse("There are currently no tasks in your list.");
+            return;
+        }
+        printResponse("Here are the tasks in your list:");
+        for (int i = 0; i < taskCount; ++i) {
+            printResponse(String.format("%d. %s", i + 1, tasks[i]));
+        }
+    }
+
+    private static void markTaskDone(String cmd) {
```

Comment:
Good use of naming and follows naming convention. It would be good to add a few comments for readability.